examples/hacker-news/storage.json

```diff
@@ -1,4 +0,0 @@
-{"type":"StoryCreated","payload":{"title":"Welcome to reSolve Hacker News!","text":"To import news from the origin (https://news.ycombinator.com/), run `npm run import` or `yarn run import` and restart the application","link":"","userId":"f6118edd-aeca-46a7-99fa-56ca72555543","userName":"resolve"},"aggregateId":"0b8aa1db-85ea-46e6-a795-597bd506494e","aggregateVersion":1,"timestamp":1522141781312,"_id":"EoY818SOcOAceAWL"}
```

Comment:
This file had been stored in repository not accidentally - it contains event with information about `import` launching. It's was necessary to prevent user from disambiguation when he launches fresh downloaded example HackerNews